spec/features/projects/show_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe "Project show page", type: :feature do
+    before(:each) do
+        @recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+        @furniture_challenge = Challenge.create(theme: "Apartment Furnishings", project_budget: 1000)
+
+        @news_chic = @recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+        @boardfit = @recycled_material_challenge.projects.create(name: "Boardfit", material: "Cardboard Boxes")
+
+        @upholstery_tux = @furniture_challenge.projects.create(name: "Upholstery Tuxedo", material: "Couch")
+        @lit_fit = @furniture_challenge.projects.create(name: "Litfit", material: "Lamp")
+
+        @jay = Contestant.create(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+        @gretchen = Contestant.create(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+        @kentaro = Contestant.create(name: "Kentaro Kameyama", age: 30, hometown: "Boston", years_of_experience: 8)
+        @erin = Contestant.create(name: "Erin Robertson", age: 44, hometown: "Denver", years_of_experience: 15)
+
+
+        ContestantProject.create(contestant_id: @jay.id, project_id: @news_chic.id)
+        ContestantProject.create(contestant_id: @gretchen.id, project_id: @news_chic.id)
+        ContestantProject.create(contestant_id: @gretchen.id, project_id: @upholstery_tux.id)
+        ContestantProject.create(contestant_id: @kentaro.id, project_id: @upholstery_tux.id)
+        ContestantProject.create(contestant_id: @kentaro.id, project_id: @boardfit.id)
+        ContestantProject.create(contestant_id: @erin.id, project_id: @boardfit.id)
+
+    end
+    describe "As a visitor" do
+        describe "When I visit a projects show page ('/projects/:id')" do
+            it 'shows projects name, material, theme of challenge that the project belongs to' do
+    
+                visit "/projects/#{@news_chic.id}"
+
+                expect(page).to have_content(@news_chic.name)
+                expect(page).to have_content(@news_chic.material)
+                expect(page).to have_content(@recycled_material_challenge.theme)
```

Comment:
Another good thing to add to this test is a `.to_not` method - meaning, we want to _only_ see attributes of the News Chic project, and nothign else. Example: 

```
expect(page).to_not have_content(@boardfit.name)
```
If we add this line to your test, it should also pass if the controller & view is set up correctly.